### PR TITLE
Add social proof component

### PR DESCRIPTION
**Pull request by grushikhin**

### CODE REVIEW

**Review by solovev — APPROVED** [review]

File: app/(landing)/content/en.mdx [hunk 1]
```diff
@@ -195,6 +195,24 @@
 
 </InDevelopment>
 
+<SocialProof>
+  <SocialProofHeader>
+    <h3> BATTLE-TESTED</h3>
```
[the remark this answers is not in the document]
Comment (LeoVS09) [reply app/(landing)/content/en.mdx:200]: For me looks good anyway. Do you want it to be "Battle-Tested"?

File: app/(landing)/content/en.mdx [hunk 2]
```diff
@@ -195,6 +195,24 @@
...
+    <h2> Most Thorough Testing of Protocol <br /> by Large Community</h2>
```
Comment (LeoVS09) [inline app/(landing)/content/en.mdx:201]:
```suggestion
    <h2>Most Thoroughly Tested Protocol <br /> by a Large Community</h2>
```

File: app/(landing)/views/social-proof/social-proof-stat-item.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const SocialProofStatItem: React.FC<StatItemProps> = ({ number, description, children }) => {
+  return (
+    <div className={styles.statItem}>
+      <div className={`${styles.number} ${interFont.className}`}>{number}</div>
+      <div className={styles.description}>{description}</div>
+      {children && <div className={styles.children}>{children}</div>}
+    </div>
+  )
+}
+
```
Comment (LeoVS09) [inline app/(landing)/views/social-proof/social-proof-stat-item.tsx:20]: In general, this is good, but in future you can implement it through nesting
```suggestion

/**
* @example
* <SocialProofStatItem>
*   <ProofStatNumber>20 000+</ProofStatNumber>
*   <ProofStatNumber>Members in community</ProofStatNumber>
*   <SocialProofLinks />
* </SocialProofStatItem>
*/
const SocialProofStatItem: React.FC<React.PropsWithChildren> = ({ children  }) => {
  return (
    <div className={styles.statItem}>
    {children}
    </div>
  )
}

export const ProofStatNumber: React.FC<React.PropsWithChildren> = ({children}) =>
 <div className={`${styles.number} ${interFont.className}`}>{children}</div>

export const ProofStatDescription: React.FC<React.PropsWithChildren> = ({children}) =>
 <div className={styles.description}>{children}</div>

```

File: app/(landing)/views/social-proof/social-proof.module.scss [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        line-height: var(--line-height-750);
+        font-weight: var(--font-medium);
+        margin-bottom: var(--margin-300);
+        color: var(--color-text-800);
+        text-align: left;
+        max-width: $main-seciton-width;
+        @media (max-width: 1000px) {
+            max-width: 100%;
+        }
+    }
```
Comment (LeoVS09) [inline app/(landing)/views/social-proof/social-proof.module.scss:43]: I think existing h2 have proper styles, no need to override them

**Review by LeoVS09 — CHANGES_REQUESTED** [review]

File: app/(landing)/content/en.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (solovev) [reply app/(landing)/content/en.mdx:200]: i suggest to add spaces around "-", i.e.: `BATTLE - TESTED`

[the remark this answers is not in the document]
Comment (grushikhin) [reply app/(landing)/content/en.mdx]: fixed

File: app/(landing)/content/en.mdx (hunk 2 above)
Comment (grushikhin) [reply app/(landing)/content/en.mdx]: applied fix

File: app/(landing)/views/social-proof/social-proof-stat-item.tsx (hunk 3 above)
Comment (grushikhin) [reply app/(landing)/views/social-proof/social-proof-stat-item.tsx:20]: Got it, might be at next update

File: app/(landing)/views/social-proof/social-proof.module.scss (hunk 4 above)
Comment (grushikhin) [reply app/(landing)/views/social-proof/social-proof.module.scss]: Applied default styles with gradient

**Review by LeoVS09 — APPROVED** [review]

**Review by LeoVS09 — APPROVED** [review]

**Pull request merged**